operator/cmd/root.go

```diff
@@ -106,9 +107,27 @@ var (
 			}
 		}),
 
+		cell.Provide(func(
+			operatorCfg *operatorOption.OperatorConfig,
+		) api.SharedConfig {
+			return api.SharedConfig{
+				EnableK8s: operatorCfg.EnableK8s,
+			}
+		}),
+
+		api.HealthHandlerCell(
+			kvstoreEnabled,
+			func() bool {
+				return IsLeader.Load()
```

Comment:
Use just `IsLeader.Load`, no need to wrap into a closure. Also does `IsLeader` need to be exported?